### ISSUE TITLE
Add first-time setup menu

### ISSUE THREAD

**Issue by programnoir** [programnoir/GDTemplate#8]
This is a first-time setup menu that attempts to accommodate users who are launching a game for the first time.

* This menu should be very simple - A more detailed settings menu comes right after.
* Language/Accessibility
  * Language and accessibility features should be offered first.
* Audio
  * Don't play music in the beginning.
  * Option to set premaster to 0%
  * Dedicated sound effect volume slider.
* Essential video features (window size/fullscreen and game scale).

Ideas for this menu that would need to be implemented separately:
* Narrated options.
* Dark or bright UI themes.
* Font selection
* Font size
